src/libstd/sys/redox/rt.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright 2016 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+//! Defintion of functions like _start for the linker
+
+use sys::syscall::exit;
+
+#[unstable(feature = "sys_rt", issue = "0")]
+#[no_mangle]
+#[naked]
+#[cfg(target_arch = "x86")]
+pub unsafe fn _start() {
```

Comment:
Hygiene error! In order to generate executables, we have to have a naked _start function at the libstd root.